auth/token-managers/jwt-token-manager.ts

```diff
@@ -82,7 +82,7 @@ export class JwtTokenManager extends TokenManager {
 
     const decodedResponse = decode(this.accessToken);
     if (!decodedResponse) {
-      const err = 'Access token recieved is not a valid JWT';
+      const err = 'Access token received is not a valid JWT';
```

Comment:
I'm always trying to score points with @pyrooka :joy:

Reply:
Oh man, the amount of point I should give you, would cause a buffer overflow in any system out there... :)